setup.py

```diff
@@ -22,7 +22,7 @@
     packages=find_packages(exclude=("tests",), where="."),
     url="https://github.com/MetroStar/conda-vendor",
     entry_points={"console_scripts": ["conda-vendor = conda_vendor.__main__:cli"]},
-    install_requires=["ruamel.yaml", "conda-lock", "click"],
+    install_requires=["ruamel.yaml", "conda-lock>=1.2.1", "click"],
     setup_requires=["wheel"],
```

Comment:
Why was conda-lock  > 1.2.1 required?

Reply:
cross platform solve didn't work with the previous version.